packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -0,0 +1,175 @@
+  --data-urlencode 'grant_type=urn:ietf:params:oauth:grant-type:device_code' \
+  --data-urlencode 'device_code={deviceCode)' \
+```
+
+Note the paramters that are being passed:
+
+* `grant_type`: Identifies the mechanism that Okta uses to retrieve the tokens. Value: `urn:ietf:params:oauth:grant-type:device_code`
+* `device_code`: The string that the device uses to exchange for an access token. Use the `device_code` value from the device verification response.
+
+**Response example**
+
+Okta returns a pending response if the user doesn't complete the authentication.
+
+```json
+  {
+     "error": "authorization_pending",
+     "error_description": "The device authorization is pending. Please try again later."
+  }
+```
+
+After the user completes the authentication process by visiting the `/activate` URL and follow the instructions on their device to get an activation code, the token is returned in the response from the `/token` endpoint.
+
+```json
+ {
+   "token_type": "Bearer",
+   "expires_in": 3600,
+   "access_token":
+      "eyJraWQ...JQuDJh8g",
+   "scope": "offline_access openid profile",
+   "refresh_token": "zcLdr1FBXwtI9ej98VVVwtjDd-SmaoL06qr_UcY2tNA",
+   "id_token": "eyJraWQ...WI6KR0aQ"
+ }
+```
+
+If you need a refresh token, ensure that you've initially requested the `offline_access` scope through the `/device/authorize` endpoint. To renew access tokens in the future, you can use the refresh token.
+
+> **Note:** The unique user code is valid for 10 minutes. The user needs to use this code and complete the authentication process within that time. If not, the unique user code expires and the device can request a new user code by using the `/device/authorize` endpoint.
+
+## Revoke the token
+
+To revoke the tokens, the smart device must make a request to the `/revoke` endpoint.
+
+```bash
+  curl --location --request POST 'https://${yourOktaDomain}/oauth2/default/v1/revoke' \
+  --header 'Accept: application/json' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  ``--data-urlencode ``'token=refresh_token' \
+  --data-urlencode 'token_type_hint=refresh_token' \
+  --data-urlencode 'client_id={client_id}' \
+```
+
+## Request Single Logout
+
+When the user signs out of an application, the application sends a `/logout` request to the Okta Authorizataion Server, which revokes the device secret.
+
+```bash
+  url --location --request GET `https://${yourOktaDomain}/oauth2/default/v1/logout` \
+  --data-urlencode `id_token_hint={id_token}` \
+  --data-urlencode `device_secret={device_secret}` \
+  --data-urlencode `post_logout_redirect_uri=https%3A%2F%2Fclient1.example.${yourOktaDomain}%2Flogout` \
+  --data-urlencode `state=2OwvFrEMTJg` \
+```
+
+The Authorization Server invalidates the ID token and refresh token that are issued for the `sid` and `device_secret`. If the invalidated refresh token is used to renew tokens, the request fails. The existing access token isn't revoked and is valid until it naturally expires.
```

Comment:
This does not look right. Where does `sid` and `device_secret` come for Device authorization grant?

Reply:
Moved to Native SSO